packages/cashc/src/ast/AstBuilder.ts

```diff
@@ -333,6 +337,16 @@ export default class AstBuilder
     return stringLiteral;
   }
 
+  createDateLiteral(ctx: LiteralContext): IntLiteralNode{
+    const rawString = (ctx.DateLiteral() as TerminalNode).text;
+    const stringValue = rawString.substring(6, rawString.length - 2);
+    const timestamp = (Date.parse(stringValue)/1000);
+    if(isNaN(timestamp)) throw new ParseError();
```

Comment:
Since this error could happen, I'd like the parse error to include a message. Something like `Incorrectly formatted date "${stringValue}"`. I noticed that `ParseError` isn't really set up very well to do this, so I updated the class so that you can do `new ParseError(errorMessage, location)`. If you rebase on top of `master` you should get this updated `ParseError`.

Reply:
Added!

Reply:
Thanks!